pymongoexplain/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.1.0.dev0'
+__version__ = '1.0'
```

Comment:
What happened to the patch version? 

This needs to be `1.0.0`.

Reply:
Oops. Fixed it